automatic/1password/tools/chocolateyInstall.ps1

```diff
@@ -3,10 +3,8 @@
 $packageArgs = @{
   packageName    = $env:ChocolateyPackageName
   fileType       = 'exe'
-  url            = 'https://c.1password.com/dist/1P/win6/1PasswordSetup-7.9.832.exe'
+  url            = 'https://downloads.1password.com/win/1PasswordSetup-latest.exe'
```

Comment:
Great work on this overall! I think we need to point directly to the version listed in the nuspec so it doesn't fail the checksum when the version updates. 1PasswordSetup-latest.exe is always the newest version but we can get the specific version URL as well. In this case it is "https://downloads.1password.com/win/1PasswordSetup-8.8.0.exe". That way the URL is tied to its checksum indefinitely. Then you can add back in the checksum. Obviously would create a version variable to use throughout versus latest.